5.x/crud-operation-list-entries.md

```diff
@@ -206,6 +206,23 @@ $this->crud->limit();
 $this->crud->orderBy();
 // please note it's generally a good idea to use crud->orderBy() inside "if (!$this->crud->getRequest()->has('order')) {}"; that way, your custom order is applied ONLY IF the user hasn't forced another order (by clicking a column heading)
 ```
+**NOTE:** we call this constrains `query constrains`, and when you filter or search we call it `table constrains`. All the `query constrains` added on the setup operation cannot be reseted by `Reset Button`, and all the `table constrains` are applied on top of the `query constrains`.
+
+#### Custom Order
+
+<a name="custom-order"></a>
+
+List operation uses the `model key`(usually **id**) as default to order entries by `DESC`. You can modify this behavior by applying an order key:
+```php
+protected function setupListOperation()
+{
+    //change default order key if not passed by datatable
+    if (!$this->crud->getRequest()->has('order')){
+        $this->crud->orderBy('updated_at', 'desc');
+    }
+}
+```
+**NOTE**: This is a `query constrain`, not a `table constrain`, so the reset button will not reset this order if applied without `has('order')` check. It becomes the `base query` for that operation, so any filters etc you add on table will be applied AFTER the `query orders/constrains` you define on your setup operation.
```

Comment:
I don't understand. Can you please rephrase so I understand? Why do we introduce these terms? Are they actually useful or do they just make it more difficult to understand?

```suggestion
**NOTE**: If you include the `has('order')` check, the reset button will be able to reset this order. Without this check, the order becomes permanent, so any filters etc you add on table will be applied _after_ the order you've defined.
```

Reply:
Pedro this was your writing, right? 😅

Reply:
Shame 👎